bodhi/server/services/updates.py

```diff
@@ -92,6 +94,12 @@ def get_update(request):
                  error_handler=bodhi.server.services.errors.html_handler)
 def get_update_for_editing(request):
     """Return a single update from an id, title, or alias for the edit form"""
+    proxy_request = bodhi.server.security.ProtectedRequest(request)
+    validate_acls(proxy_request)
+    # If validate_acls produced 0 errors, then we can edit this update.
+    can_edit = len(proxy_request.errors) == 0
+    if not can_edit:
+        raise HTTPForbidden()
```

Comment:
@ryanlerch I *think* that one is actually using that to determine if the user should be presented with the edit buttons, for which the Pyramid ACL system wouldn't help I don't think. I'm not 100% sure, but if that is true I think that usage is probably OK.